### ISSUE TITLE
DEPRECIATION Warning

### ISSUE THREAD

**Issue by nsharrok** [kellyselden/ember-i18n-inject#8]
Using Ember 3.1 I'm getting
```
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: mrx -> ember-i18n-inject -> ember-cli-babel
```
I checked
```
npm ls ember-cli-babel@5
mrx@12.9.0 C:\clients\MRSolutions\MRx\mrx
+-- ember-i18n-inject@0.0.2
| `-- ember-cli-babel@5.2.8
```

Can you upgrade please?

**Comment by kellyselden** [comment]
I'm not using this anymore, but feel free to submit a fix.

**Comment by kellyselden** [comment]
Should be fixed now in v1.0.0

**Comment by nsharrok** [comment]
Thank you Kelly
